lib/baseClasses/HttpError.js

```diff
@@ -90,7 +90,7 @@ function HttpError() {
      * @property
      * @type {Function}
      */
-    if (opts.hasOwnProperty('toJSON')) {
+    if (Object.prototype.hasOwnProperty.call(opts, 'toJSON')) {
```

Comment:
Why the conversion to `.call`?

Reply:
It is the autofix of following eslint rule [`no-prototype-builtins`](https://eslint.org/docs/rules/no-prototype-builtins), that got added into `eslint:recommended` config